mandatory/2-function-creation.js

```diff
@@ -16,7 +22,15 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+if (num > 100) {
+  num = 100
```

Comment:
According to the test you need to add extra check for numbers that does not have decimal places before you round them up so, if the number already has decimal places round it up to 2place, if it does not have it then just return it as string with the percentage sign append it to the end like so:
23 => "23%
23.3939 => "23.38%"